test/Serilog.Sinks.PeriodicBatching.Tests/PeriodicBatchingSinkTests.cs

```diff
@@ -112,5 +122,13 @@ public void WhenAnEventIsEnqueuedItIsWrittenToABatch_FlushWhileRunning()
             Assert.True(bs.IsDisposed);
             Assert.False(bs.WasCalledAfterDisposal);
         }
+
+        [Fact]
+        public void SubclassesCanBeConstructedUsingNoQueueLimitConstant()
+        {
+#pragma warning disable 618
+            var _ = new NullBatchedSink(batchSizeLimit: 100, TimeSpan.FromSeconds(2), queueLimit: PeriodicBatchingSink.NoQueueLimit);
```

Comment:
This is allowed by compiler since [7.2](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.2/non-trailing-named-arguments), so we can either fix this line or remove `install` section in `appveyor.yml` to make use of the latest sdk / compiler